translations/ui-marccat/en.json

```diff
@@ -49,6 +49,8 @@
   "search.associated.bibliographic": "Associated bibliographic records",
   "search.history.title": "Recent Searches",
   "browse.results.title": "Access Point",
+  "browse.results.see": "See",
+  "browse.results.seefrom": "Seen From",
```

Comment:
Update this label to be `seenfrom`. Otherwise, looks good.

Reply:
ok, very attentive! done